src/models/TreeData.ts

```diff
@@ -47,7 +47,7 @@ export class TreeDataSource {
      * 
      * @param root warning: will be mutated and returned as ITreeDataSource
      */
-    constructor(input: TreeNode | TreeDataSource | Array<TreeNode>) {
+    constructor(input: TreeNode | TreeDataSource | Array<any>) {
         if (this.isDataSource(input)) {
             this.nodesById = input.nodesById;
```

Comment:
@vinkobedek 
Will this be handled if any doesnt have this fields? (will the TreeGrid know how to work if these fields are undefined)

Reply:
as disscused,  changed to to any to avoid confusion.